style/js/shield_defs.js

```diff
@@ -183,6 +183,32 @@ export function loadShields(shieldImages) {
     },
   };
 
+  shields["US:KY"] = circleShield("white", "black");
+  shields["US:KY:AA"] = {
+    backgroundImage: shieldImages.shield40_us_ky_parkway,
+    textColor: "#003f87",
+    padding: {
+      left: 2,
+      right: 2,
+      top: 2,
+      bottom: 6,
+    },
+  };
+  shields["US:KY:Parkway"] = Object.assign(
+    {
+      refsByWayName: {
+        "Audobon Parkway": "AU",
+        "Bluegrass Pkwy": "BG",
+        "Cumberland Pkwy": "LN",
+        "Hal Rogers Pkwy": "HR",
+        "Mountain Pkwy": "MP",
+        "Purchase Pkwy": "JC",
```

Comment:
I extracted these initialisms from KYTC’s internal alphanumeric route numbers. Some of them are pretty recognizable, but some like “JC” are more obscure. (It’s from the namesake in the official name.) For the benefit of non-locals who don’t remember the older, more usable shields, I’d be open to changing it to something more intuitive, like “P”, but such ad-hoc abbreviations definitely shouldn’t go into OSM.